drivers/SmartThings/matter-switch/src/init.lua

```diff
@@ -454,7 +474,7 @@ local function do_configure(driver, device)
     end
   end
 
-  if profile_name then
+  if not device:get_field(IS_AQARA_SWITCH_DEVICE) and profile_name then
```

Comment:
I think I'd prefer leaving the configure logic as-is, and add a more general SWITCH_DEVICE_PROFILED with an early return to the top of the function. This SWITCH_DEVICE_PROFILED field can be set for all devices using initialize_switch, not just aqara devices.
 
```suggestion
local function do_configure(driver, device)
  if device:get_field(SWITCH_DEVICE_PROFILED) then
     return
  end
 local energy_eps = embedded_cluster_utils.get_endpoints(device, clusters.ElectricalEnergyMeasurement.ID)
  ...
  if profile_name then
```

Reply:
Yes, that's a good idea. However, when `energy_eps` or `power_eps` have a value greater than `0`, there is always a switch endpoint, so `BUTTON_DEVICE_PROFILED` was used instead of `SWITCH_DEVICE_PROFILED`.

Reply:
Sorry, I don't see any reference to BUTTON_DEVICE_PROFILED in the PR. Have you made this update?

Reply:
Yes, I did it at the commit below.
`1d680a035ac0c89035afc21fa6ec4fec8b896da2`

Reply:
yes I see, sorry about that, the commit was not showing up on my UI without toggling something.